CHANGELOG.md

```diff
@@ -4,6 +4,50 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.22.0] - 2022-06-20
+
+With this release, subxt can perform runtime upgrades to ensure that extrinsics are properly constructed for the
```

Comment:
What does "perform runtime upgrades" mean? It makes it sound like `subxt` hosts a runtime and can upgrade itself like substrate. Maybe we can rephrase this?